docusaurus/docs/cloud/cli/cloud-cli.md

```diff
@@ -41,13 +41,18 @@ Deploy a new local project (< 100MB) in Strapi Cloud.
 strapi deploy
 ```
 
-This command must be used after the `login` one. It deploys a local Strapi project on Strapi Cloud, without having to host it on a remote git repository beforehand. The terminal will inform you when the project is successfully deployed on Strapi Cloud.
+This command must be used after the `login` one. It deploys a local Strapi project on Strapi Cloud, without having to host it on a remote git repository beforehand.
 
-Once the project is first deployed on Strapi Cloud with the CLI, the `deploy` command can be reused to trigger a new deployment of the same project.
+:::note
```

Comment:
Thank you for the PR, @gonbaum!

Where is this new note supposed to end? 🤔

Currently we're missing a closing `:::` tag on a new line, which triggers rendering issues as seen in the screenshot.
![Screenshot 2024-11-12 at 17 52 55](https://github.com/user-attachments/assets/5bc2e827-42f0-48ef-9016-3d9bc1b836cc)

Reply:
Yes, sorry about that. I'm not familiar with the markdown for documentation, so yes, the note should end where you mention in the comment below.